src/formatDate.js

```diff
@@ -9,6 +9,72 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const separatorFrom = fromFormat.pop();
+  const separatorTo = toFormat.pop();
```

Comment:
Using `pop` to remove the last element from `fromFormat` and `toFormat` arrays directly modifies the input arrays, which is generally not a good practice as it can lead to side effects. Consider using a different approach that doesn't modify the input arrays, such as accessing the last element by index.